plugins/language_gmi.lua

```diff
@@ -0,0 +1,22 @@
+-- mod-version:2 -- lite-xl 2.0
+local syntax = require "core.syntax"
+
+
+
+syntax.add {
+  name = "Gemtext",
+  files = { "%.gmi$" },
+  patterns = {
+    { pattern = { "```", "```" },           type = "string"   },
+    { pattern = "#.-",                      type = "keyword"  },
+    { pattern = "%*.*",                     type = "keyword2" },
```

Comment:
This is just my preference, but I think that we should only highlight the `*`, not the whole line.

Reply:
my thinking behind highlighting the entire line for bullet lists was for it to match how headings and quotes are handled (the entire line is highlighted in those cases), but highlighting just the `*` does improve readability, so i'll change that accordingly.